src/args/flag_handlers/tokenInfo.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Handles the token info from Groq.
+ *
+ * @param {string[]} args - The arguments passed to the CLI.
+ * @returns {boolean} True if the token usage flag is present, otherwise false.
+ */
+function handleTokenFlag(args: string[]): boolean {
+  try {
```

Comment:
suggestion: the try/catch block here is unnecessary, because no error is going to be thrown.

This should be something like:

```typescript
function handleTokenFlag(args: string[]): boolean {
return args.includes('--token-usage') || args.includes('-tu');
}
```